aws_lambda_builders/workflows/go_modules/validator.py

```diff
@@ -47,7 +56,10 @@ def validate(self, runtime_path):
         if p.returncode == 0:
             out_parts = out.decode().split()
             if len(out_parts) >= 3:
-                version_parts = [int(x.replace("rc", "")) for x in out_parts[2].replace(self.LANGUAGE, "").split(".")]
+                version_parts = [
+                    int(GoRuntimeValidator.sanitize_go_version_part(version_part))
```

Comment:
Could we use this regex `go(\d)\.(x|\d+)` to extract major/minor versions instead?
We could directly compare it to expected values.
We could also use it at line 50 if we put it into a `get_go_versions()` function, returning both values.